src/platforms/common/performance/instrumentation/custom-instrumentation.mdx

```diff
@@ -36,6 +36,16 @@ To capture transactions customized to your organization's needs, you must first
 
 </Note>
 
+<PlatformSection supported={["native"]}>
+
+<Alert level="info" title="Important">
+
+This is currently an experimental feature. Performance monitoring is still a work-in-progress feature, and its API may be unstable. It may also have bugs. Feedback is appreciated, and should be submitted as an issue in the appropriate repository.
```

Comment:
same?

Reply:
pointing to https://github.com/getsentry/sentry-docs/pull/4640#discussion_r792867942 just for completeness.

Reply:
"This is currently an experimental feature. Performance monitoring is still a work-in-progress feature […]" is a bit repetitive, I would suggest condensing these two statements.